### DISCUSSION TITLE
Support Linux?

### DISCUSSION THREAD

**Discussion by rootkit-io** [vpnhood/VpnHood#45] [Q&A] [answered]
I use linux *arch will it's for pc too????

**Comment by trudyhood** [comment, marked as answer]
Hi, You can run VpnHood Server on linux but running VpnHood Client on linux is not supported yet.

**Reply by Heus-Sueh** [reply]
do you have any date to support the client on linux? or is the client never going to be supported on linux?

**Reply by trudyhood** [reply]
While we don't currently have plans to develop VpnHood Client on Linux, we believe it would be a valuable addition to our product offering. Implementing the IPacketCapture interface is all that's needed to make this a reality, and we welcome contributions from anyone who is interested in helping to make it happen. Thank you for your interest in our product.
